src/lib/core/datetime/native-date-adapter.ts

```diff
@@ -31,7 +31,7 @@ const DEFAULT_DAY_OF_WEEK_NAMES = {
 
 /** Creates an array and fills it with values. */
 function range<T>(length: number, valueFunction: (index: number) => T): T[] {
-  return Array.apply(null, Array(length)).map((v: undefined, i: number) => valueFunction(i));
+  return Array.apply(null, Array(length)).map((_v: undefined, i: number) => valueFunction(i));
```

Comment:
We could eliminate this case by switching to a `for` loop... (which is faster anyway)